docs/index.html

```diff
@@ -7,56 +7,49 @@
+            If there's a topic that you think we should have docs for, please
+            consider <a href="../contribute">contributing</a>.
         </p>
         <p>
             <a href="https://github.com/ev3dev/ev3dev/issues/153#issuecomment-93602107">Some pages</a>
-            have not been migrated from our old wiki yet. We'd love some help with getting these last few pages migrated:
-            if you would like to help out, read more about what's needed in the <a href="../contribute#write-some-documentation">documentation section</a>
+            have not been migrated from our old wiki yet. We'd love some help
+            with getting these last few pages migrated! If you would like to
+            help out, read more about what's needed in the
+            <a href="../contribute#write-some-documentation">documentation section</a>
             of our "contributing" page.
         </p>
     </div>
 </div>
 
 <h2>System setup</h2>
 <p>
-    If you're looking to set up a new installation of ev3dev, either on an EV3 brick or
-    <a href="../news/2015/05/13/raspberry-pi-image-release">on a Raspberry Pi</a>, check out
-    our <a href="getting-started">getting started</a> guide. Once you have a working installation
-    of ev3dev, you'll probably want to try out your new software. To help people learn how to
-    use ev3dev, we maintain a set of <a href="tutorials/">tutorials</a> that walk you through
-    everything from imaging an SD card to using specialized hardware.
+    If you're looking to set up a new installation of ev3dev, check out
+    our <a href="/docs/getting-started">Getting Started</a> guide.
+</p>
+
+<h2>Tutorials</h2>
+<p>
+    We have a page of tutorials which cover a wide range of topics, from
+    networking to specialized hardware configuration. Head over to our
+    <a href="/docs/tutorials">Tutorials page</a> to see the full list.
 </p>
 
 <h2>Writing some software</h2>
 <p>
-    To take advantage of ev3dev's main functionality, you'll probably need to write some code
-    or download and compile preexisting software. The easiest way to do this is to explore the
-    <a href="libraries">available libraries</a> that have been developed to simplify developing
-    with ev3dev. If you need to directly program against the ev3dev driver APIs, you can read our
-    info pages on the following topics:
+    To take advantage of ev3dev's main functionality, you'll probably want to
+    use a pre-built library from <a href="/docs/libraries">our central list of
+    known libraries</a>. Choose a language to get started!
 </p>
 
-<ul>
-    <li>
-        <a href="ports">Input and output ports</a>
-    </li>
-    <li>
-        <a href="sensors">Supported Sensors and Other Input Devices</a>
-    </li>
-    <li>
-        <a href="motors">Supported Motors and Other Output Devices</a>
-    </li>
-    <li>
-        <a href="drivers">ev3dev kernel drivers</a>
-    </li>
-</ul>
+<h3>Low-level driver access</h3>
```

Comment:
I don't think of hardware drivers as writing software. I refer to those pages often just to get technical info on sensors.